### PR TITLE
[ID] Update `Tournaments`

### PR DESCRIPTION
**Pull request by FazlyMR**
This pull request is meant to majorly update the Indonesian translation of the `Tournaments` article, adding new tournament instalments as well as new tournament series from the original English version of the above article.

In addition, a single mention on official tournament support was removed to match the English version.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

(Shouldn't need to be reviewed by a fluent speaker, as it is an update to an existing list without any wording changes.)

### CODE REVIEW

**Comment by Nivalyx** [comment]
hmm, iirc there are more things that are also needs to be updated in the page rather than just the listing tho (then again the entire `id.md` for this page is already very old and outdated by now lol)

with that in mind it would be fitting if the pr can be renamed into sth like "[ID] Update tournaments listing in `Tournaments`" ig, but as for the update to the listing by itself i checked it and everything looks fine

**Comment by FazlyMR** [comment]
> Hmm, iirc there are more things that need updating here than just the listing tho. As it stands the `id.md` is already lagging *very* far behind the original `en.md` so yeah...
> 
> With that in mind it would be better to rename the PR into sth like `[ID] Update tournaments listing in Tournaments` to make it more representative of what it is I guess, but as for the proposed update by itself everything looks fine from what I see

Got it, PR title changed as suggested. Also, a mention about official tournament support was removed recently after checking the English article but other than that there's nothing more.

**Review by Nivalyx — DISMISSED** [review]
yup, i think it's fine to push this for now as is and update the other details later as a follow-up in a different pr

**Comment by FazlyMR** [comment]
A new tournament not yet in the English version has been added with a wikilink to a folder that does not yet exist as of time of writing.

File: wiki/Tournaments/id.md [hunk 1]
```diff
@@ -376,6 +450,9 @@ Turnamen/Kompetisi tidak resmi yang diselenggarakan oleh komunitas.
...
 | [Asian Taiko Showdown 2018](AOTS/ATS_2018) | 2018-06-02 | 2018-07-08 | ::{ flag=TW }:: [Smallwu](https://osu.ppy.sh/users/2512120) | ::{ flag=JP }:: [nyanmi-1828](https://osu.ppy.sh/users/6866480) | ::{ flag=JP }:: [butter0414](https://osu.ppy.sh/users/7935022) |
 | [Asian-Oceanian Taiko Showdown](AOTS/AOTS_1) | 2019-08-09 | 2019-09-15 | ::{ flag=TW }:: [Smallwu](https://osu.ppy.sh/users/2512120) | ::{ flag=JP }:: [Saikoro](https://osu.ppy.sh/users/741819) | ::{ flag=HK }:: [MTDex](https://osu.ppy.sh/users/9468283) |
+| [Indonesian Taiko Showdown 2](AOTS/IDTS_2) | 2021-12-03 | 2022-01-09 | ::{ flag=ID }:: [Servatory](https://osu.ppy.sh/users/4013317) | ::{ flag=ID }:: [freezebear](https://osu.ppy.sh/users/1943301) | ::{ flag=ID }:: [Veltlion](https://osu.ppy.sh/users/10999079) |
+| [Indonesian Taiko Beginner Showdown](AOTS/ITBS_1) | 2022-04-16 | 2022-06-05 | ::{ flag=ID }:: [salym](https://osu.ppy.sh/users/19089549) | ::{ flag=ID }:: [WeebReen](https://osu.ppy.sh/users/10129901) | ::{ flag=ID }:: [N\_jaaayy](https://osu.ppy.sh/users/17787564) |
+| [Indonesian Taiko Beginner Showdown 2](AOTS/ITBS_2) | 2023-06-03 | 2023-07-16 | *TDB* | *TDB* | *TDB* |
```
Comment (Walavouchey) [inline wiki/Tournaments/id.md:455]: there will be a broken link if we add this now (meaning this pr can't be merged as is). please add it when the article exists

```suggestion
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/id.md [hunk 2]
```diff
@@ -1,7 +1,3 @@
----
-outdated_translation: true
----
```
Comment (Walavouchey) [inline wiki/Tournaments/id.md:3]: > > Hmm, iirc there are more things that need updating here than just the listing tho. As it stands the `id.md` is already lagging \*very\* far behind the original `en.md` so yeah...

@Nivalyx if the translation is still outdated even after these changes, then this marker shouldn't be removed

Comment (Nivalyx) [reply wiki/Tournaments/id.md:3]: o yes true

@FazlyMR

Comment (FazlyMR) [reply wiki/Tournaments/id.md:3]: >yup, i think it's fine to push this for now as is and update the other details later as a follow-up in a different pr

I'm still not sure what that is lacking in the Indonesian translation. I have checked it with the current English version and it seems to not be lagging very far aside from needing to update the tournament listing.

Comment (Nivalyx) [reply wiki/Tournaments/id.md:3]: mainly some missing lines every here and there, for example this line is still not present in the current Indonesian translation 
image

File: wiki/Tournaments/id.md [hunk 3]
```diff
@@ -1,7 +1,3 @@
...
-
 # Turnamen
 
 Laman ini diperuntukkan untuk turnamen-turnamen yang ada di osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [pada tautan berikut](https://osu.ppy.sh/community/forums/55).
```
Comment (FazlyMR) [inline wiki/Tournaments/id.md:3]: Maybe I suggest adding the following line to make it up-to-date.

```suggestion
Laman ini diperuntukkan untuk turnamen-turnamen yang ada di osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [pada tautan berikut](https://osu.ppy.sh/community/forums/55).

Silahkan kunjungi halaman [dukungan resmi turnamen] untuk informasi mengenai cara mengajukan permintaan dukungan resmi untuk turnamenmu (lencana, *screening process*, dll.).
```

File: wiki/Tournaments/id.md (hunk 2 above)
Comment (FazlyMR) [reply wiki/Tournaments/id.md:3]: Ohh, I didn't see that. Ok then. See my next comment for my suggestion on that.

File: wiki/Tournaments/id.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Tournaments/id.md:3]: sorry for my late reply but yup

File: wiki/Tournaments/id.md [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
 
+Silahkan kunjungi halaman [dukungan resmi turnamen](/wiki/Tournaments/Official_support) untuk informasi mengenai cara mengajukan permintaan dukungan resmi untuk turnamenmu (lencana, *screening process*, dll.).
```
Comment (Nivalyx) [inline wiki/Tournaments/id.md:5]:
```suggestion
Silakan kunjungi halaman [dukungan resmi turnamen](/wiki/Tournaments/Official_support) untuk memperoleh informasi seputar cara mengajukan permintaan dukungan resmi untuk turnamenmu (lencana, proses *screening*, dll.).
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**